llvm-external-projects/iree-dialects/lib/Dialect/LinalgTransform/IR/StructuredTransformOpsExt.cpp

```diff
@@ -1268,13 +1274,28 @@ reductionCallback(transform_ext::MatchCallbackResult &res, Location loc,
     return DiagnosedSilenceableFailure::success();
   return emitSilenceableFailure(loc) << "failed to match";
 }
+static DiagnosedSilenceableFailure
```

Comment:
Can we have a basic wrapping mechanism that just forwards and adds true? 
This way we don't need to declare a linear number of these trivial indirection functions in the future.